cassandra/src/test/groovy/io/micronaut/configuration/cassandra/CassandraConfigurationSpec.groovy

```diff
@@ -35,86 +39,78 @@ class CassandraConfigurationSpec extends Specification {
 
         expect: "No beans are created"
         !applicationContext.containsBean(CassandraConfiguration)
-        !applicationContext.containsBean(Cluster)
+        !applicationContext.containsBean(CqlSession)
 
         cleanup:
         applicationContext.close()
     }
 
     void "test single cluster connection"() {
         given:
-        // tag::single[]
```

Comment:
These tags link up to the documentation; are you planning to replace them / edit the docs to better illustrate new usage?

Reply:
oh, didn't realize that. I guess I'll have to add that back in

Reply:
Not a problem! Thanks :)